packages/atomic/src/utils/props-utils.ts

```diff
@@ -1,12 +1,44 @@
 import {isArray} from '@coveo/bueno';
 import {ComponentInterface, getElement} from '@stencil/core';
+import {ReactiveElement} from 'lit';
 import {camelToKebab, kebabToCamel} from './utils';
 
 interface MapPropOptions {
   attributePrefix?: string;
   splitValues?: boolean;
 }
 
+export function mapProperty<Element extends ReactiveElement>(
```

Comment:
I see that we're testing the `mapAttributesToProp` function, but we're not testing `mapProperty`. Shouldn't we?

Reply:
the decorator is essentially calling